tools/mason/MasonPublish.chpl

```diff
@@ -267,6 +267,22 @@ proc dryRun(username: string, registryPath : string, isLocal : bool) throws {
     }
   }
   else {
+    const spacer = '------------------------------------------------------';
+    writeln('Checking Registry with ' + registryPath + ' path.');
+    var registryTest = registryPathCheck(registryPath, username, false);
+    writeln(spacer);
+    writeln('The current mason environment is:');
+    returnMasonEnv();
+    var regExists: bool = false;
+    const regUrl = "https://github.com/chapel-lang/mason-registry";
+    for regs in MASON_REGISTRY {
```

Comment:
I think we can simplify this still:

```chpl
var reg = MASON_REGISTRY;
if reg.contains(('mason-registry'), "https://github.com/chapel-lang/mason-registry")) {
  writeln('   In order to use a local registry, ensure that MASON_REGISTRY includes the path.');
}
```

Reply:
I get an error on using this refactor:
```
./MasonPublish.chpl:238: In function 'dryRun':
./MasonPublish.chpl:279: error: unresolved call 'list(2*string,false).contains("mason-registry", string)'
$CHPL_HOME/modules/internal/ChapelArray.chpl:1747: note: this candidate did not match: _domain.contains(i: _value.idxType ...rank)
./MasonPublish.chpl:279: note: because method call receiver with type list(2*string,false)
$CHPL_HOME/modules/internal/ChapelArray.chpl:1122: note: is passed to formal 'this: _domain'
$CHPL_HOME/modules/internal/ChapelError.chpl:363: note: candidates are: TaskErrors.contains(type t)
$CHPL_HOME/modules/internal/ChapelRange.chpl:739: note:                 range.contains(ind: idxType)
note: and 5 other candidates, use --print-all-candidates to see them
```